### ISSUE TITLE
no-uninitialized rule false positive

### ISSUE THREAD

**Issue by NN---** [palantir/tslint#3680]
### Bug Report

- __TSLint version__: 5.9.1
- __TypeScript version__: 2.6.2
- __Running TSLint via__: (pick one) gulp

#### TypeScript code being linted

```ts
class A {
 public readonly x: () => string;
 
 constructor() {
  try { 
   x = () => "A";
  } catch (e) {
  }
 }
}
```

with `tslint.json` configuration:

```json
{"rules": {
"no-uninitialized": [true, "properties"]
}}
```

#### Actual behavior
Error about uninitialized 'x'.

#### Expected behavior
No error since it is initialized.

**Comment by ajafff** [comment]
> no-uninitialized is not a TSLint core rule. You should bug the author of tslint-strict-null-checks

Duplicate of #3570

**Comment by NN---** [comment]
@ajafff My bad. Sorry.
